treeherder/model/migrations/0021_investigatedtests.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.0.8 on 2020-08-14 07:18
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0020_textlogerror_job'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='InvestigatedTests',
+            fields=[
+                ('id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('test', models.CharField(max_length=255)),
```

Comment:
is this long enough?  I would think we have some edge cases

Reply:
Good point, yeah.  Maybe 350 would be better?

Reply:
I have made it a textfield because the corresponding things in the jobType model were TextField
Should I change it

Reply:
Sure, textfield sounds good.